src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DocumentPresentation/TextDocumentUriPresentationEndpoint.cs

```diff
@@ -91,7 +99,7 @@ protected override IRazorPresentationParams CreateRazorRequestParameters(UriPres
                 return null;
             }
 
-            var componentTagText = await TryGetComponentTagAsync(razorFileUri, cancellationToken).ConfigureAwait(false);
+            var componentTagText = await TryGetComponentTagAsync(documentContext, cancellationToken).ConfigureAwait(false);
```

Comment:
This still needs reverting, the razorFileUri is the file being dropped, but documentContext is the file being dropped onto